src/fidesops/models/datasetconfig.py

```diff
@@ -117,12 +118,15 @@ def _convert_dataset_field_to_graph(field: FidesopsDatasetField) -> Field:
                     reference.dataset, ref_collection, ".".join(ref_fields)
                 )
                 references.append((address, reference.direction))
+        if meta_section.length is not None:
```

Comment:
This check isn't doing anything. The result is the same if we just have `length = meta_section.length` like below, without the check.

Reply:
This check will still pass along `0` values right? `0` is not `None`, so length will still be assigned in that case. The check for truthiness is `if meta_section.length:`

Reply:
Thought a bit more about this, and now I think I should leave it, with perhaps a comment - 

Reason is that this method should pass along a value if it is a valid int. If there’s a 0 passed in, that’s not the same as a None being passed in and it should get passed through to the Field object creation. So, we pass the value in if value is not None
In a way this is kind of moot since in the validations I’m explicitly rejecting any values <= 0, but I think that in the future it’s possible to imagine some kind of weird use case where we’d actually want to allow you to set a 0 length (maybe you want empty strings? ) and if we decided we wanted to support that, we could just change the validation and we wouldn’t have this weird buried filter that wouldn’t let the 0's go through in another part of the code.